pkg/scheduler/controller/shoot/reconciler.go

```diff
@@ -345,14 +345,14 @@ func filterCandidates(shoot *gardencorev1beta1.Shoot, versionedShootList []garde
 }
 
 // getSeedWithLeastShootsDeployed finds the best candidate (i.e. the one managing the smallest number of shoots right now).
-func getSeedWithLeastShootsDeployed(seedList []gardencorev1beta1.Seed, shootList []gardencorev1beta1.Shoot) (*gardencorev1beta1.Seed, error) {
+func getSeedWithLeastShootsDeployed(seedList []gardencorev1beta1.Seed, versionedShootList []gardencorev1beta1.Shoot) (*gardencorev1beta1.Seed, error) {
```

Comment:
Nit: Revert this change, it is evident that these are versioned objects since it's in a controller.